### ISSUE TITLE
Unexpected cascaded replication for conflicting items using PostgreSQL 9.5 with pglogical 1.1.2

### ISSUE THREAD

**Issue by sireon** [2ndQuadrant/pglogical#40]
Having **two** PostgreSQL nodes running on _5590_ and _5591_ ports respectively. Both nodes are providers and also each one is subscribing to the other one. One simple table to be replicated: `public.products ( product_no integer PRIMARY KEY, name text, price numeric )`.

The attached shell script ([test_conflicts_with_disabled_subscriptions.txt](https://github.com/2ndQuadrant/pglogical/files/494049/test_conflicts_with_disabled_subscriptions.txt)) is doing essentially the same thing as explained below in order to reproduce the issue.

**Steps to reproduce the issue:**

Disable both subscriptions on both nodes (trying to simulate a network fragmentation here but this will also fail with stopped PostgreSQL nodes)
Having the same row on both nodes, update that row (with the same unique id).
Re-enable both subscriptions and check the replication

**Expected results:**
The last update should win and the same value should be observed on both nodes

**What happened instead:**
In a first phase, a proper replication can be observed. However, after few seconds, there is a subsequent replication attempt that will generate a segmentation fault on one of the nodes (and the database is in recovery mode not being able to respond to sql queries). 

Note: The process will end up switching the values between nodes. Sometimes, the results are unpredictable and both nodes end up with the first (oldest) update 

Logs on _5590_:

```
LOG:  CONFLICT: remote UPDATE on relation public.products. Resolution: apply_remote.
LOG:  starting logical decoding for slot "pgl_pglogicaldemo_node_5590_subscrip0fb5853"
DETAIL:  streaming transactions committing after 0/1782080, reading WAL from 0/1781094
LOG:  logical decoding found consistent point at 0/1781094
DETAIL:  There are no running transactions.
LOG:  could not receive data from client: Connection reset by peer
LOG:  unexpected EOF on standby connection
WARNING:  terminating connection because of crash of another server process
DETAIL:  The postmaster has commanded this server process to roll back the current transaction and exit, because another server process exited abnormally and possibly corrupted shared memory.
HINT:  In a moment you should be able to reconnect to the database and repeat your command.
ERROR:  connection to other side has died
LOG:  worker process: pglogical apply 16384:2489372982 (PID 3471) exited with exit code 1
LOG:  starting logical decoding for slot "pgl_pglogicaldemo_node_5590_subscrip0fb5853"
DETAIL:  streaming transactions committing after 0/1782080, reading WAL from 0/1781094
LOG:  logical decoding found consistent point at 0/1781094
DETAIL:  There are no running transactions.
LOG:  could not receive data from client: Connection reset by peer
LOG:  unexpected EOF on standby connection
LOG:  starting logical decoding for slot "pgl_pglogicaldemo_node_5590_subscrip0fb5853"
DETAIL:  streaming transactions committing after 0/1782080, reading WAL from 0/1781094
LOG:  logical decoding found consistent point at 0/1781094
DETAIL:  There are no running transactions.
```

Logs on _5591_:

```
LOG:  CONFLICT: remote UPDATE on relation public.products. Resolution: keep_local.
LOG:  worker process: pglogical apply 16384:263554356 (PID 3499) was terminated by signal 11: Segmentation fault
LOG:  terminating any other active server processes
WARNING:  terminating connection because of crash of another server process
DETAIL:  The postmaster has commanded this server process to roll back the current transaction and exit, because another server process exited abnormally and possibly corrupted shared memory.
HINT:  In a moment you should be able to reconnect to the database and repeat your command.
LOG:  all server processes terminated; reinitializing
LOG:  database system was interrupted; last known up at 2016-09-26 19:26:12 UTC
LOG:  recovered replication state of node 1 to 0/1781EF8
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/1782494
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled

```

**Comment by ringerc** [comment]
Interesting issue. We don't really view multimaster as fully supported by pglogical at this stage, but crashes are certainly not expected. Can you get a backtrace of the crash? Enable core dumps, compile pglogical against a postgresql built with --enable-debug or one with debuginfo packages installed, run the test, and when it crashes find the core file, attach gdb, and run 'bt'.

**Comment by PJMODOS** [comment]
I think this is same issue as #31 which was fixed in 1.2.
